homework-submission/week2/src/Counter/CounterManager.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+
+export default class CounterManager extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
```

Comment:
this.state doesn't have `isChecked`!? how can you use it however it is not defined!?

Reply:
But I want to take initial value from App.js. How can I pass initial values to checkboxes.